zebra/rtadv.c

```diff
@@ -178,9 +178,12 @@ static void rtadv_send_packet(int sock, struct interface *ifp)
 		/* XXX Free on shutdown. */
 		adata = malloc(CMSG_SPACE(sizeof(struct in6_pktinfo)));
 
-		if (adata == NULL)
+		if (adata == NULL) {
 			zlog_err(
 				"rtadv_send_packet: can't malloc control data");
+			exit(-1);
+		} else
+			memset(adata, 0, CMSG_SPACE(sizeof(struct in6_pktinfo)));
```

Comment:
I think it would be easier to use `calloc()` instead of `malloc()` + `memset()`

Reply:
Correct, fixed